stl/inc/bit

```diff
@@ -233,6 +233,32 @@ _NODISCARD int _Checked_arm_arm64_countl_zero(const _Ty _Val) noexcept {
 }
 #endif // defined(_M_ARM) || defined(_M_ARM64)
 
+_NODISCARD constexpr unsigned short _Byteswap_ushort(const unsigned short _Val) noexcept {
+    if (_STD is_constant_evaluated()) {
+        return static_cast<unsigned short>((_Val << 8u) | (_Val >> 8u));
+    } else {
+        return _byteswap_ushort(_Val);
+    }
+}
+
+_NODISCARD constexpr unsigned long _Byteswap_ulong(const unsigned long _Val) noexcept {
+    if (_STD is_constant_evaluated()) {
+        return (_Val << 24u) | ((_Val << 8u) & 0x00FF0000ul) | ((_Val >> 8u) & 0x0000FF00ul) | (_Val >> 24u);
+    } else {
+        return _byteswap_ulong(_Val);
+    }
+}
+
+_NODISCARD constexpr unsigned long long _Byteswap_uint64(const unsigned long long _Val) noexcept {
+    if (_STD is_constant_evaluated()) {
+        return (_Val << 56) | ((_Val << 40) & 0x00FF'0000'0000'0000ull) | ((_Val << 24) & 0x0000'FF00'0000'0000ull)

```

Comment:
Consistency:
* shift counts here are `int`s, vs. `unsigned int` for the other two overloads
* bit masks here have digit separators every 4 hexits, whereas there are no separators in the first two overloads
It would be nice to be consistent across all three overloads. (I'd prefer `int` for shift counts and digit separators, but won't reject anything consistent.)